lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -228,7 +228,99 @@ public function get_item( $request ) {
 			);
 		}
 
-		return $font_face;
+		return $response;
+	}
+
+	/**
+	 * Creates a font face for the parent font family.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function create_item( $request ) {
+		// Create the font face post so we have an ID to attach the font files to.
+		$font_face_id = wp_insert_post(
+			array(
+				'post_type'   => $this->post_type,
+				'post_parent' => $request['parent'],
+				'post_status' => 'publish',
+				'post_title'  => 'Font Face',
```

Comment:
Maybe we could use something more meaningful here as the `fontFamily` value?

Reply:
Idea: use post_name for font family

Reply:
Done, now using `fontFamily` for post_title, and a generated slug from that for post_name.